docs/medical-api/more-info/webhooks.mdx

```diff
@@ -284,3 +282,52 @@ The format follows:
 
   </Expandable>
 </ResponseField>
+
+#### Medical Record
+
+When querying for a Patient's consolidated data, you can also request this information as a pdf or html.
+The following is the result of the [previous section](#patient-consolidated-data), but with the `Bundle` containing a document reference to the medical record that was converted from the requested FHIR bundle.
+
```

Comment:
nit: let's be explicit here and say that the document can be downloaded from the `url` in the first item in the `content` array in the returned `DocumentReference`